doc/examples/reconst_ivim.py

```diff
@@ -128,13 +128,15 @@
 $\mathbf{f}$).
 """
 
-ivimmodel = IvimModel(gtab)
+ivimmodel = IvimModel(gtab, fit_method='LM')
```

Comment:
It would be good to also add an example of using the MIX method into this example and maybe compare the two methods to each other.